### PR TITLE
Gui

### PR DESCRIPTION
**Pull request by AImani2**

### CODE REVIEW

**Comment by AImani2** [comment]
Hi Professor,
I am struggling to figure out the proper layout here.
Is mine correct?

**Comment by AImani2** [comment]
What component will display my images in the center of my panel?

File: src/main/java/imani/rijks/RijksFrame.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    JButton searchButton = new JButton("Search");
+    JTextField searchBar = new JTextField(42); // Set the preferred width
+    int pageNum = 1;
+
+    JPanel north = new JPanel(new BorderLayout());
+    JPanel center = new JPanel(new GridLayout(0, 5)); // why this?
+    // create components for the pictures and add them to center
+
+    ApiKey apiKey = new ApiKey();
+    RijksService service = new RijksServiceFactory().getService();
```
Comment (andrewoid) [inline src/main/java/imani/rijks/RijksFrame.java]: These can all be private

File: src/main/java/imani/rijks/Main.java [hunk 2]
```diff
@@ -4,4 +4,34 @@ public class Main {
     // usage
     // ApiKey apiKey = new ApiKey();
     // String keyString = apiKey.get();
+
+    // ...
+//
+//    You will need to create a RijksService that has 3 methods.
```
Comment (andrewoid) [inline src/main/java/imani/rijks/Main.java]: You don't need this file anymore.

File: src/main/java/imani/rijks/RijksFrame.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+            public void mouseClicked(MouseEvent e) {
+                pageNum--;
+                update();
+            }
+        });
+
+        searchButton.addMouseListener(new MouseAdapter() {
+            @Override
+            public void mouseClicked(MouseEvent e) {
+                update();
```
Comment (andrewoid) [inline src/main/java/imani/rijks/RijksFrame.java:74]: set the pageNum to 0

Comment (andrewoid) [reply src/main/java/imani/rijks/RijksFrame.java:74]: I mean 1. 
And you didn't fix this.

**Pull request merged**